crates/fsrt/src/main.rs

```diff
@@ -1,9 +1,14 @@
 #![allow(clippy::type_complexity)]
+
+mod test;
```

Comment:
```suggestion
#[cfg(test)]
mod test;
```

Conditionally including the test module will speed up build times, as the test functionality will only get compiled if you're actually running `cargo test`.

Reply:
Fixed. Thank you!